.github/workflows/benchmark.yml

```diff
@@ -52,31 +57,32 @@ jobs:
       env:
         CC: gcc
       run: |
-        . ./activate
         git clone https://github.com/Chia-Network/clvm_tools.git --branch=main --single-branch
         python -m pip install ./clvm_tools
         python -m pip install colorama
         maturin develop --release -m wheel/Cargo.toml
 
     - name: python mypy
       run: |
-        . ./activate
-        mypy tests/test_*.py
+        mypy tests
 
     - name: python tests
       run: |
-        . ./activate
         pytest tests
 
     - name: test generators
+      if: matrix.os == 'ubuntu-latest'
+      # TODO: This should happen everywhere but the benchmarking part is not passing
+      #       on all platforms so this can be revisited separately.
```

Comment:
we should probably run this on our dedicated benchmarking runners

Reply:
The benchmark aspect I suspect should be made optional, probably default off, and it should be run with that on the benchmarks runners, sure.  But I wouldn't want to lose regular test coverage of it and would like coverage across platforms to be added here.  Anyways, I'm good with leaving this for another PR.  My next round with our workflows might be some sort of reusable library mechanism to help with the cross-repo duplication and remaining variances.